error.yaml

```
@@ -0,0 +1,24 @@
+description: API error
+properties:
+  message:
+    description: A descriptive error message
+    type: string
+    example: |-
+      Validation error: the value provided for `my_input` is a number but
+      should be a string.
+  expected_object_response:
+    description: |-
+      What the endpoint would have returned if the request was successful.
+    type: string
+    example: my_object
```

This would potentially need updating in a few places in the other repo, but is this a class name? If so, might we use the following example?

```suggestion
    example: MyObject
```

---

It's whatever would have gone in the `data_type` field if the request was successful. Personally, I've been putting everything JSON related as snake case - ex `item` or `item_alias` but can change this practice.

---

Maybe my question should have been "What should go in the `data_type` field?"

In any case, I think you're right about the snake case thing 👍